.husky/pre-push

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-npx --no-install validate-branch-name
+make prettier-fix
```

Comment:
Always before a commit will be fixed by prettier, right @jkergal ?

Reply:
That's not a real problem that the branch naming isn't correct until we push it on remote. So for speed and performance we verify it only before pushing it. :)

Reply:
Nice